hotxlfp/formulas/utils.py

```diff
@@ -8,22 +8,24 @@
 import operator
 from . import error
 import datetime
+import torch
 import time
 from dateutil.parser import parse as to_date
 
 DEFAULT = lambda: 0
 
 OPERATOR_DICT = {
-    '+': operator.add,
-    '-': operator.sub,
-    '*': operator.mul,
-    '/': operator.truediv,
-    '>': operator.gt,
-    '<': operator.lt,
-    '<>': operator.ne,
-    '=': operator.eq,
-    '>=': operator.ge,
-    '<=': operator.le,
+    "+": operator.add,
+    "-": operator.sub,
+    "*": operator.mul,
+    "/": operator.truediv,
+    ">": operator.gt,
+    "<": operator.lt,
+    "<>": operator.ne,
+    "=": operator.eq,
+    ">=": operator.ge,
+    "<=": operator.le,
+    "^": lambda a, b: torch.pow(torch.tensor(a), torch.tensor(b)),
```

Comment:
should you use operator.pow?

eventually we may want to operate on our own symbol objects instead of torch tensors, at that point using pow would be preferable because we can do operator overloading